maven-extension/src/main/java/io/opentelemetry/maven/handler/GoogleJibBuildHandler.java

```diff
@@ -86,8 +86,8 @@ public void enrichSpan(SpanBuilder spanBuilder, ExecutionEvent executionEvent) {
     spanBuilder.setAttribute(
         MavenOtelSemanticAttributes.MAVEN_BUILD_CONTAINER_REGISTRY_URL,
         "https://" + registryHostname);
-    spanBuilder.setAttribute(SemanticAttributes.HTTP_URL, "https://" + registryHostname);
-    spanBuilder.setAttribute(SemanticAttributes.HTTP_METHOD, "POST");
+    spanBuilder.setAttribute(SemanticAttributes.URL_FULL, "https://" + registryHostname);
```

Comment:
I propose to decouple and solve later the question of adopting the new convention and to implement it aligning with the OTel Java Auto Instr that added a config flag `OTEL_SEMCONV_STABILITY_OPT_IN` / `otel.semconv-stability.opt-in`.
I created a PR narrowed down to the change of lib:
* https://github.com/open-telemetry/opentelemetry-java-contrib/pull/1059